static/img/teaser/sources.txt

```diff
@@ -0,0 +1,9 @@
+All screenshots taken from random winners TT Screenshot of the Month competitions
```

Comment:
This reads like a bad sentence :D
`All screenshots are taken from random winners of the TT Screenshot of the Month competition.`
?